src/server.js

```diff
@@ -174,16 +174,13 @@ app.use(function*(next) {
       .then(response => (this.body = response))
       .catch(error => (renderError = error));
   } else {
+    let clipInfo = this.request.query.fullPage === "1" ? {fullPage: true} : { clipInfo: { x: 0, y: 0, width: width, height: height}} ;
```

Comment:
The key in the object with clip rect should be named `clip`, as it was passed to `.screenshot()` before.

Also, this package currently supports all the way down to node v6.0.0 which doesn't support object spread used below. It would be nice to not have to bump required node version all the way to node v8.6.0 just for that one line. Conversely `Object.assign` is easy to use to extend objects with more properties and is supported way before node v6 even. Granted, node v6 is EoL April 1st 2019, so before that we'll likely bump to v8 LTS.